nservicebus/operations/tuning.md

```diff
@@ -12,37 +12,30 @@ tags:
  - performance
  - throttling
  - concurrency
-reviewed: 2016-11-15
+reviewed: 2018-10-10
 ---
 
-NServiceBus uses defaults that ensure good performance in most common cases. While this is usually the preferred mode of operation there are situations where tuning needs to be applied.
-
+NServiceBus uses defaults that ensure good performance in common cases. While this is usually the preferred mode of operation there are situations where tuning might be desired.
 
 ## Tuning concurrency
 
-
 partial: defaults
 
+Define a maximum concurrency setting that will make sure that no more messages than the specified value are ever being processed at the same time. Set this value to `1` to process messages sequentially. If not specified the transport will choose an optimal value.
```

Comment:
>If not specified the transport will choose an optimal value

Do we need to specify what the transport optimizes for? (maximum throughput while not running out of resources)